sphinx/parsers.py

```diff
@@ -53,7 +53,7 @@ def set_application(self, app):
         self.info = app.info
 
 
-class RSTParser(docutils.parsers.rst.Parser, Parser):
+class RSTParser(docutils.parsers.rst.Parser):
```

Comment:
The Parser inheritance was added in an earlier commit of this PR (#4360), but is now not needed